specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json

```diff
@@ -0,0 +1,731 @@
+{   
+    "swagger": "2.0",
+    "info": {
+        "title": "InstanceMetadataClient",
+        "description": "The Azure Instance Metadata Client",
+        "version": "2018-10-01"
+    },
+    "host": "169.254.169.254",
```

Comment:
the host is usually a qualified name. I don't expect stable api to be called via ip address.

Reply:
This is how our service is used; that is, the user will make a request to http://169.254.169.254. For example, cloud init queries our service as follows:

open 'http://169.254.169.254/metadata/reprovisiondata?api-version=2017-04-02' with {'method': 'GET', 'url': 'http://169.254.169.254/metadata/reprovisiondata?api-version=2017-04-02', 'timeout': 1.0, 'allow_redirects': True, 'headers': {'Metadata': 'true', 'User-Agent': 'Cloud-Init/18.4-0ubuntu1~16.04.2'}} configuration